geode-core/src/test/java/org/apache/geode/cache/query/partitioned/PRInvalidQueryDUnitTest.java

```diff
@@ -12,102 +12,83 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-
 package org.apache.geode.cache.query.partitioned;
 
 import static org.apache.geode.cache.query.Utils.createPortfolioData;
-import static org.junit.Assert.*;
+import static org.apache.geode.test.dunit.Host.getHost;
+import static org.apache.geode.test.dunit.Invoke.invokeInEveryVM;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 import org.apache.geode.cache.query.data.PortfolioData;
-import org.apache.geode.internal.cache.PartitionedRegionDUnitTestCase;
-import org.apache.geode.test.dunit.Host;
-import org.apache.geode.test.dunit.LogWriterUtils;
 import org.apache.geode.test.dunit.VM;
-import org.apache.geode.test.dunit.cache.internal.JUnit4CacheTestCase;
-import org.apache.geode.test.dunit.internal.JUnit4DistributedTestCase;
+import org.apache.geode.test.dunit.cache.CacheTestCase;
 import org.apache.geode.test.junit.categories.DistributedTest;
 
 @Category(DistributedTest.class)
-public class PRInvalidQueryDUnitTest extends PartitionedRegionDUnitTestCase
-
-{
-  /**
-   * constructor *
-   *
-   * @param name
-   */
-
-  public PRInvalidQueryDUnitTest() {
-
-    super();
-  }
+public class PRInvalidQueryDUnitTest extends CacheTestCase {
 
-  public void setCacheInVMs(VM... vms) {
-    for (VM vm : vms) {
-      vm.invoke(() -> PRQueryDUnitHelper.setCache(getCache()));
-    }
-  }
+  private static final String name = "Portfolios";
 
-  int totalNumBuckets = 100;
+  private static final int i = 0;
+  private static final int step = 20;
+  private static final int cnt = 0;
+  private static final int cntDest = 90;
+  private static final int redundancy = 0;
 
-  PRQueryDUnitHelper prq = new PRQueryDUnitHelper();
+  private final PRQueryDUnitHelper prq = new PRQueryDUnitHelper();
 
-  final String name = "Portfolios";
+  private VM vm0;
+  private VM vm1;
+  private VM vm2;
+  private VM vm3;
 
-  final int i = 0, step = 20, cnt = 0, cntDest = 90;
+  @Before
+  public void setUp() {
+    vm0 = getHost(0).getVM(0);
+    vm1 = getHost(0).getVM(1);
+    vm2 = getHost(0).getVM(2);
+    vm3 = getHost(0).getVM(3);
+    setCacheInVMs(vm0, vm1, vm2, vm3);
+  }
 
-  final int redundancy = 0;
+  @After
+  public void tearDown() {
+    disconnectAllFromDS();
+    invokeInEveryVM(() -> PRQueryDUnitHelper.setCache(null));
+  }
 
   /**
-   * This test <pr> 1. Creates PR regions across with scope = DACK <br>
-   * 2. Creates a Local region on one of the VM's <br>
-   * 3. Puts in the data in PR region & the Local Region <br>
-   * 4. Queries the PR qith an Invalid Query Syntax <br>
-   * 5. Verfies that there is an QueryInavalidException
+   * 1. Creates PR regions across with scope = DACK
```

Comment:
Delete extraneous blank lines that were added to javadocs unless they help readability (these do not).